Add Arch AUR link, and add X1 Yoga 3 as a supported device

jmarmstrong1207 opened the pull request:
This script is useful to me, so I decided to create a package for it in the Arch AUR. Plus, I added my laptop that I use it for.

alesguzik commented:
Nice! Thanks :)

Regarding the PKGBUILD there's an issue that it specifies the license as GPL, but this project is actually licenced under MIT.

As a strong suggestion, `$USER` should be treated as undefined during packaging. Packages are often built in a sandbox, so `$USER` might be anything really. Or just think that someone might build your PKGBUILD under `sudo`. The right thing would be to copy an example config into e.g. `/etc/detect-tablet-mode/watch_tablet.yml.example` and then, after you built and installed the package, copy or symlink the example into your home directory.

Also, from https://wiki.archlinux.org/index.php/Arch_package_guidelines#Directories :

> Packages should not contain any of the following directories: 
> * ...
> * /home

jmarmstrong1207 commented:
Ah, I forgot about not using /home/. I fixed it, changed the license, and it now copies the config file to /etc/. I also added a message for instructions on what to do.

alesguzik commented:
👍
